.github/workflows/ci.yml

```
@@ -0,0 +1,101 @@
+name: CI
+'on':
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  # lint-dockerfile:
```

Maybe is a good idea to explain the reason of the commented block (already in the PR body but not inside the code in the repo). For instance:

```suggestion
  # FIXME: hadolint won't work since the docker file has not been updated.
  # lint-dockerfile:
```